src/pages/Details/components/QuantityBox.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './QuantityBox.scss';
+
+export const QuantityBox = props => {
+  const { count, setCount } = props;
+  const handleClick = num => {
+    return setCount(prev => prev + num);
```

Comment:
```suggestion
    setCount(prev => prev + num);
```
반환할 값이 없기 때문에 return 은 작성 안하셔도 됩니다!